cclient/response/get_balances.h

```diff
@@ -17,16 +17,26 @@ extern "C" {
 static const UT_icd ut_uint64_icd UTARRAY_UNUSED = {sizeof(uint64_t), NULL,
                                                     NULL, NULL};
 typedef struct {
-  UT_array* balances;
   int milestoneIndex;
```

Comment:
Not part of this PR but can you switch this to snake_case ?

Reply:
sure